Server/MirObjects/PlayerObject.cs

```diff
@@ -7672,7 +7672,7 @@ private void ProtectionField(UserMagic magic)
 
             int duration = 45 + (15 * magic.Level);
             int maxAC = (int)Math.Round(Stats[Stat.MaxAC] * (0.2 + (0.03 * magic.Level)));
-            int minAC = (int)Math.Round(Stats[Stat.MinAC] * (0.2 + (0.03 * magic.Level)));
+            int minAC = (int)Math.Round(Stats[Stat.MaxAC] * (0.2 + (0.03 * magic.Level)));
```

Comment:
Why is minAC getting MaxAC stat?

Reply:
The skill Rage's MinAC increase should be the same as MaxAC.
The code has been changed to a better way.